webauthn/helpers/parse_authentication_credential.py

```diff
@@ -0,0 +1,21 @@
+from typing import Union
+from pydantic import ValidationError
+
+from .exceptions import InvalidAuthenticationResponse
+from .structs import PYDANTIC_V2, AuthenticationCredential
+
+
+def parse_authentication_credential(json_val: Union[str, bytes, bytearray]) -> AuthenticationCredential:
```

Comment:
I think this method would better communicate its functionality if it was `parse_authentication_credential_json()` instead. Without the `_json` it almost reads like you'd pass an instance of `AuthenticationCredential` into it, but of course this is just for parsing JSON into a `AuthenticationCredential`.

Reply:
I agree. Fixed.